package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-kubeless",
-  "version": "0.3.1",
+  "version": "0.4.0",
```

Comment:
should we bump that up to 1.x.x so that the serverless templates download it automatically....otherwise the templates will still break.

Reply:
That is something in theory desirable. We are updating the major version since the new version contains breaking changes. In other case, if someone, without updating Kubeless, creates a new template it won't work. The problem is that we depend on `serverless` releases so if we want to avoid that we can just bump the patch `0.3.2`